README.md

```diff
@@ -33,58 +31,79 @@ This guide will install CSI plugin in the *kube-system* namespace of Kubernetes
   
   * Check installed
   
-  ```
-  $ qbd -v
-  Package Version:       2.0.4-cb3daa5-190821224030-centos75
-  Loaded Module Version: 2.0.4-cb3daa5-190821224030-centos75
-  NeonSAN Static Library Version: 2.1.14-83d762a`
-  ```
+    ```
+    $ qbd -v
+    Package Version:       2.0.4-cb3daa5-190821224030-centos75
+    Loaded Module Version: 2.0.4-cb3daa5-190821224030-centos75
+    NeonSAN Static Library Version: 2.1.14-83d762a`
+    ```
+
+
+- Download installation files 
   
-- Download installation file 
-```
-$ wget https://raw.githubusercontent.com/yunify/qingstor-csi/master/deploy/neonsan/kubernetes/releases/csi-neonsan-v1.1.0.yaml
-```
+  ```
+  $ wget https://github.com/bmingithub/qingstor-csi/raw/master/deploy/neonsan/kubernetes/csi-neonsan-canary.tar.gz
```

Comment:
We should not put binary, compressed file into source code repo.
For testing, we can develop a script to generate the binary or tar files.
For released version, we can put those file into asserts of release page, such as https://github.com/yunify/qingstor-csi/releases/tag/v0.3.0.